app/helpers/display_format_helper.rb

```diff
@@ -0,0 +1,23 @@
+module DisplayFormatHelper
+  def date_format(date)
+    DateTime.parse(date).strftime("%B %Y") unless date.nil?
+  end
+
+  def head_info_list_maker(info)
+    [info['format'].humanize, date_format(info['public_timestamp'])]
+  end
+
+  def link_format(link)
+    if link.start_with?("https://", "http://")
+      link
+    elsif link.start_with?("www.")
+      "https://#{link}"
+    else
+      "https://gov.uk" + link
+    end
+  end
+
+  def name(link)
+    link_format(link).gsub(%r{https?://[^/]+/}, '').tr('-', ' ').tr('/', ' / ')
```

Comment:
I did this after we decided that the relative path was more useful then the title for working out whether a document was meant to be there based on the search. I will look at the URI module, thanks :)